app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_user_session_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_user_registration_path(resource_name) %><br />
```

Comment:
- [ ] This error was introduced after the last reviewer's suggestions so he could not catch it. When I click on the sign up link on the log in page I get this error. Kindly fix it. I had to manually type in the correct path in the browser to visit the registration page.

  ![image](https://github.com/ahmidou09/blog_app/assets/36542986/0604ccd4-46b5-4466-924f-e12b15846b51)